plugin/pki/path_venafi_cert_enroll.go

```diff
@@ -270,7 +270,7 @@ func (b *backend) pathVenafiCertObtain(ctx context.Context, req *logical.Request
 	if !role.NoStore {
 		if role.StoreBy == storeByCNString {
 			//Writing certificate to the storage with CN
-			b.Logger().Debug("Putting certificate to the certs/" + reqData.commonName)
+			b.Logger().Debug(fmt.Sprintf("Putting certificate to the certs/" + reqData.commonName))
```

Comment:
minor wording (cosmetic) observation: could be written as "Writing certificate" instead of "Putting certificate"